server/superdesk/io/rss.py

```diff
@@ -307,6 +310,9 @@ def _create_package(self, text_item, image_items):
         """
         package = {
             'type': 'composite',
+            'guid': 'package:' + text_item['guid'],
```

Comment:
IMO we should use this: https://github.com/superdesk/superdesk/blob/master/server/apps/archive/common.py#L88